src/utils/alerts.js

```diff
@@ -24,6 +24,16 @@ export const dispatchWithAlert = (action, data) => store.dispatch({
 
 export const actionsPending = (types) => (typeof types === 'string' ? [types] : types).some((type) => store.getState().status?.actionStatus[type]?.pending)
 
+export const actionsPendingByPrefix = (typePrefix) => {
+    const { actionStatus = {} } = store.getState().status
+
+    return Object.keys(actionStatus).some((type) => {
+        if (type.includes(typePrefix)) {
```

Comment:
I think this should be `startsWith` -- or for backwards compatibility, maybe `.slice(typePrefix.length) ===`, to avoid false positive matching other actions that have the same string somewhere else in their action name.

Reply:
Right